tests/code-check-roave-backward-compatibility-check-on-pr/test.env

```diff
@@ -0,0 +1,2 @@
+GITHUB_EVENT_PATH=/github/workspace/GITHUB_EVENT_CONTENTS.json
+GITHUB_EVENT_NAME=pull_request
```

Comment:
For context, used environment variables come from 

https://github.com/actions/toolkit/blob/bc4be505973a6a7344bfd71e1b32f77e1755310c/packages/github/src/context.ts#L28-L53

@Ocramius should other variables be set to some non-empty dummy value?

Reply:
At this stage, doesn't look like it.